common/datavalues/src/arrays/arithmetic.rs

```diff
@@ -200,7 +207,16 @@ where
 {
     type Output = Result<DataArray<T>>;
 
+    // Test sql: pc :) select sum(number % 5)  FROM numbers_mt(10000000000);
     fn rem(self, rhs: Self) -> Self::Output {
+        // modulus_scalar is not faster
```

Comment:
Do we still need these codes ?

Reply:
I'd like to keep these for future refactor plan.